assets/search/ico-download.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="13px" height="13px" viewBox="0 0 13 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
+    <title>ico-download</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Artboard-3" transform="translate(-29.000000, -12.000000)" fill="#999999">
+            <path d="M37.0294118,18.5 L37.0294118,14.6764706 L33.9705882,14.6764706 L33.9705882,18.5 L32.0588235,18.5 L35.5,22.7058824 L38.9411765,18.5 L37.0294118,18.5 Z M35.5,25 C39.0898509,25 42,22.0898509 42,18.5 C42,14.9101491 39.0898509,12 35.5,12 C31.9101491,12 29,14.9101491 29,18.5 C29,22.0898509 31.9101491,25 35.5,25 Z M35.5,23.4705882 C38.2451801,23.4705882 40.4705882,21.2451801 40.4705882,18.5 C40.4705882,15.7548199 38.2451801,13.5294118 35.5,13.5294118 C32.7548199,13.5294118 30.5294118,15.7548199 30.5294118,18.5 C30.5294118,21.2451801 32.7548199,23.4705882 35.5,23.4705882 Z" id="ico-download"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
svg images can be optimised with [svgo(mg)](https://jakearchibald.github.io/svgomg/) for example. Although they're very small, they can be smaller